### ISSUE TITLE
Clean-up code and make cleaner text for screen reader

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5586]
Due to https://github.com/OpenSlides/openslides-client/pull/5582 and https://github.com/OpenSlides/openslides-client/pull/5577 
I realised that the code can be a bit cleaner:

For testing: 
- Appearance in Autopilot should not have changed
- Accesibility: If you are not allowed to change your presence status, the screen reader message should be easier to understnad now
